src/main/java/org/jenkinsci/plugins/buildtriggerbadge/IconFinder.java

```diff
@@ -79,6 +79,7 @@ private String internalFindForClass(Class clazz)
 		defineIconForCause("com.cloudbees.plugins.flow.FlowCause", "flow-cause.png");
 		defineIconForCause("com.cloudbees.jenkins.plugins.BitBucketPushCause", "bitbucket.png");
 		defineIconForCause("hudson.plugins.git.GitStatus$CommitHookCause", "git-hook-cause.png");
+		defineIconForCause("org.jenkinsci.plugins.urltrigger", "url-trigger-cause.png");
```

Comment:
Oops. Yeah. Good point.

I'll fix that tomorrow and update the PR.

On Tuesday, 25 August 2015, Baptiste Mathus <notifications@github.com>
wrote:

> In src/main/java/org/jenkinsci/plugins/buildtriggerbadge/IconFinder.java
> <https://github.com/jenkinsci/buildtriggerbadge-plugin/pull/16#discussion_r37904427>
> :
>
> > @@ -79,6 +79,7 @@ private String internalFindForClass(Class clazz)
> >  		defineIconForCause("com.cloudbees.plugins.flow.FlowCause", "flow-cause.png");
> >  		defineIconForCause("com.cloudbees.jenkins.plugins.BitBucketPushCause", "bitbucket.png");
> >  		defineIconForCause("hudson.plugins.git.GitStatus$CommitHookCause", "git-hook-cause.png");
> > +		defineIconForCause("org.jenkinsci.plugins.urltrigger", "url-trigger-cause.png");
>
> Hi! thanks for the PR
>
> Your addition seems to be pointing to a package instead of a *Cause*
> implementation. Which is unlikely to work.
>
> Just read the code of the URLTrigger plugin, IIUC what you may want
> instead is the org.jenkinsci.plugins.urltrigger.URLTriggerCause class.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/buildtriggerbadge-plugin/pull/16/files#r37904427>
> .
>


-- 

--
Lucy M. H. Davies